lib/tests/validation.test.js

```diff
@@ -86,30 +86,23 @@ const testCases = [
     required: true,
   },
   {
-    fieldType: "richText",
```

Comment:
@bryan-robitaille  I removed this type of validation because the `richText` component does not appear to be something the user interacts with.